mlir-tensorrt/test/Dialect/TensorRTRuntime/one-shot-bufferize.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: mlir-tensorrt-opt %s -split-input-file -empty-tensor-to-alloc-tensor -plan-bufferize | FileCheck %s
+// RUN: mlir-tensorrt-opt %s -verify-diagnostics -allow-unregistered-dialect -split-input-file -empty-tensor-to-alloc-tensor -plan-bufferize | FileCheck %s
```

Comment:
What unregistered dialect is being used? Seems like that flag shouldn't be necessary

Reply:
Yes, it is not required. I have removed it.